### PR TITLE
LPS-117573 Fix broken Poshi tests

### PR DESCRIPTION
**Pull request by carolmariaabb**

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:forms

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsJumpToPage.testcase [hunk 1]
```diff
@@ -6,6 +6,11 @@ definition {
 	property portal.upstream = "true";
 	property testray.main.component.name = "Forms";
 
+	var fieldNameChooseYourDestiny = "";
+	var fieldNameDestiny1 = "";
+	var fieldNameDestiny2 = "";
+	var fieldNameDestiny3 = "";
+
 	setUp {
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsJumpToPage.testcase:14]: Not working!

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by brunofariasdeo** [comment]
Just started reviewing :)

**Comment by carolmariaabb** [comment]
ci:test:forms

**Comment by carolmariaabb** [comment]
Closed [#43](https://github.com/brunofariasdeo/liferay-portal/pull/43).<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**